compiler/src/iree/compiler/Codegen/Common/FoldTensorExtractOp.td

```diff
@@ -15,6 +15,6 @@ include "mlir/IR/PatternBase.td"
 // Canonicalize unnecessary tensor_load when the load is used just for
 // an extract
 def : Pat<(Tensor_ExtractOp (Bufferization_ToTensorOp $value), $indices),
-          (LoadOp $value, $indices)>;
+          (LoadOp $value, $indices, ConstBoolAttrFalse)>;
```

Comment:
@nicolasvasilache @MaheshRavishankar: in https://reviews.llvm.org/D142616, the `memref.load` op got an extra field. The default was set to false so I've added that here. Tensor.extract does not have this field: https://mlir.llvm.org/docs/Dialects/TensorOps/#tensorextract-mlirtensorextractop

Any thoughts or leave as is?

Reply:
I have no context on that. @grypp is the author of the patch. He might know. I think you should be able to leave as is?

Reply:
Looks right to me.
I have a PR that I do the same thing. 
https://github.com/iree-org/iree/pull/12062/files#diff-3132f19dbc60430a5a64dda6865dde964b121a9766a82202bd95dd1693c72dd2R18

Reply:
Awesome thanks